src/service/variable-source.js

```diff
@@ -164,6 +204,10 @@ export class VariableSource {
    */
   set(varName, syncResolver) {
     dev().assert(varName.indexOf('RETURN') == -1);
+    if (!this.isWhitelisted_(varName)) {
```

Comment:
I think this is not needed, given you have already done the whitelist filtering in `buildExpr_`